controllers/metrics.go

```diff
@@ -26,6 +28,20 @@ type DRPolicySyncMetrics struct {
 	DRPolicySyncInterval prometheus.Gauge
 }
 
+type SyncDurationMetrics struct {
+	LastSyncDuration prometheus.Gauge
+}
+
+type SyncDataBytesMetric struct {
+	LastSyncDataBytes prometheus.Gauge
+}
+
+type DRPCMetrics struct {
+	SyncMetrics
+	SyncDurationMetrics
+	SyncDataBytesMetric
+}
```

Comment:
Why do we need to embed SyncMetrics into DRPCMetrics? Can we replace SyncMetrics
with the new type, or add more fields to SyncMetrics?

Reply:
So the metrics from the DRPC are initialized as soon as the DRPC instance is created. There was only SyncMetrics which was being initialized and set before, but now we are adding two more metrics to DRPC. Instead of adding two more types to the DRPC instance, I have created a struct(DRPC Metrics) and added all the metrics to it, and passed the pointer in all the functions. in other words `SyncMetrics`, `SyncDurationMetrics`, and `SyncDataBytesMetric` are all different metrics in one struct which is easier to pass or add more metrics in the future to the DRPC instance.